src/modules/ROOT/pages/changelog/202303/05.json

```diff
@@ -0,0 +1,161 @@
+{
+  "meta": {
+    "releasedOn": "2023.08.11"
+  },
+  "added": [
+    {
+      "what": [
+        "option to configure whether module field value is on the same row as it's label."
+      ],
+      "why": [
+        "to provide the user with the option to set the position of a field value."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1339"
+      ]
+    },
+    {
+      "what": [
+        "case sensitive search in a record list if MySQL database is used."
+      ],
+      "why": [
+        "to make searching of records easier by returning similar results to the searching criteria."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1355"
+      ]
+    }
+  ],
+  
+  "changed": [
+    {
+      "what": [
+        "the visibility of the inline record editor option in a record list to hidden if a module isn't provided."
```

Comment:
this would sound more like a fix to me since (I'm guessing) it would raise an error otherwise